v21.2/install-cockroachdb-linux.md

```diff
@@ -158,6 +158,7 @@ true
 
 <div id="build-from-source-linux" class="install-option">
   <h3>Build from Source</h3>
+  <p>Consult the <a href="https://wiki.crdb.io/wiki/spaces/CRDB/pages/181338446/Getting+and+building+CockroachDB+from+source">Wiki</a> for specific, up-to-date information.</p>
```

Comment:
Given this change, rather than duplicate instructions, I think we should keep this sentence only and remove the rest. I'd suggest revising this sentence slightly to:

```
<p>See the <a href="https://wiki.crdb.io/wiki/spaces/CRDB/pages/181338446/Getting+and+building+CockroachDB+from+source">public wiki</a> for guidance.</p>
```